app/models/server/models/LivechatInquiry.js

```diff
@@ -228,6 +232,48 @@ export class LivechatInquiry extends Base {
 
 		this.remove(query);
 	}
+
+	getAbandonedQueuedItems(date) {
```

Comment:
IMO we should rename this method since we are not just talking about abandoned chats because the end-user may not have abandoned, actually the end-user was there waiting for assistance that never came.

Reply:
Yup, agreed with this.